release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -904,6 +999,16 @@ The following OpenShift CLI (`oc`) commands were removed with this release:
 * `oc adm migrate legacy-hpa`
 * `oc adm migrate storage`
 
+[id="ocp-4-11-monitoring-grafana-component-removed"]
+==== Grafana component removed from monitoring stack
+The Grafana component is no longer a part of the {product-title} {product-version} monitoring stack.
+As an alternative, browse to *Observe* -> *Dashboards* in the {product-title} web console to view monitoring dashboards.
```

Comment:
Verb choice:

I see `go to` more often than `browse` think it'd read better to reword to:

`As an alternative, go to *Observe* -> *Dashboards*...`

Reply:
`go` is better -- and shorter! Will make the change.